Alignment/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ target_include_directories(
         $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
 )
 
-target_link_libraries(ActsAlignment PUBLIC ActsCore)
+target_link_libraries(ActsAlignment PUBLIC Acts::Core)
 
 install(
```

Comment:
Would it not make sense to put some of these common operations behind the `acts_add_library(...)` function as well? I know not everybody agrees, but I myself strongly believe that this greatly simplifies the maintenance of the R&D build configuration.

I guess you already looked, but have a look at `vecmem_add_library(...)`. (https://github.com/acts-project/vecmem/blob/main/cmake/vecmem-functions.cmake) That performs a few different things that are done in the same way for the "installed vecmem libraries".

Reply:
I'm on the fence with the `install` directives (see other comment), but I think in terms of other properties, dependencies, I think I'd rather keep the `acts_add_library` function as minimal as possible, even if it means duplicating cmake instructions in a number of places.

Reply:
We use different target paths for the installed headers (`Acts`, `ActsExamples`) for the same reason we kept the namespaces separate (as discussed above), so I can't easily put them in the function without adding extra arguments